config/config.go

```diff
@@ -192,6 +192,14 @@ var (
 	DefaultExemplarsConfig = ExemplarsConfig{
 		MaxExemplars: 100000,
 	}
+
+	DefaultTSDBConfig = TSDBConfig{
+		Retention: &TSDBRetentionConfig{
+			Time:                   model.Duration(360 * time.Hour),
```

Comment:
Nit: Just to make it more readable :)
```suggestion
			Time:                   model.Duration(15 * 24 * time.Hour),
```